src/adapter/src/catalog/open.rs

```diff
@@ -1480,19 +1420,16 @@ impl Catalog {
             "we should be re-creating every dropped object"
         );
         for id in migration_metadata.all_drop_ops.drain(..) {
-            self.state.drop_item(id);
-            self.drop_plans_and_metainfos(id);
+            state.drop_item(id);
         }
```

Comment:
It looks like we lost the call to `drop_plans_and_metainfos(...)`, which I think is okay since we never re-populated them here so we weren't actually migrating anything?

Reply:
My understanding is that these plan maps are cached plans for the catalog. We were never populating these in `Catalog::open`, but instead these got populated in `Coordinator::bootstrap`. So I think that `drop_plans_and_metainfos` was just there for defensive programming, but never actually did anything.